recipes/pyface/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "5.0.0" %}
+
+package:
+  name: pyface
+  version: {{ version }}
+
+source:
+  fn: pyface-{{ version }}.tar.bz2
+  url: https://pypi.python.org/packages/source/p/pyface/pyface-{{ version }}.tar.bz2
+  md5: 0468d6a44adba0b70ec5171d1af9350a
+  patches:
+    - pyface_fix.patch
+    # fix for issue https://github.com/enthought/pyface#188 following the
```

Comment:
`#188` -> `issues/188`

Also, could we please have a one line summary about the bug? This should give someone looking at this patch some basic idea of what it does without digging any further.